test/common/stats/thread_local_store_test.cc

```diff
@@ -1816,5 +1923,83 @@ TEST_F(HistogramThreadTest, ScopeOverlap) {
   store_->histogramFromString("histogram_after_shutdown", Histogram::Unit::Unspecified);
 }
 
+size_t getMemoryUsage() {
+#ifndef WIN32
+  std::ifstream status("/proc/self/status", std::ios_base::in);
```

Comment:
note that there is already source/common/memory/stats.h -- will that do what you want?

I'm skeptical this will work on all non-windows platforms. Apple? Mobile?

Reply:
removed this.